qiskit_experiments/library/characterization/analysis/t1_analysis.py

```diff
@@ -67,3 +67,69 @@ def _evaluate_quality(self, fit_data: curve.CurveFitResult) -> Union[str, None]:
             return "good"
 
         return "bad"
+
+
+class T1KerneledAnalysis(curve.DecayAnalysis):
+    r"""A class to analyze T1 experiments with kerneled data.
+
+    # section: see_also
+        qiskit_experiments.curve_analysis.standard_analysis.decay.DecayAnalysis
+
+    """
+
+    @classmethod
+    def _default_options(cls) -> Options:
+        """Default analysis options."""
+        options = super()._default_options()
+        options.curve_drawer.set_options(
+            xlabel="Delay",
+            ylabel="Normalized Projection on the Main Axis",
+            xval_unit="s",
+        )
+        options.result_parameters = [curve.ParameterRepr("tau", "T1", "s")]
+        options.update_options(normalization=True)
+
+        return options
+
+    def _evaluate_quality(self, fit_data: curve.CurveFitResult) -> Union[str, None]:
+        """Algorithmic criteria for whether the fit is good or bad.
+
+        A good fit has:
+            - a reduced chi-squared lower than three
+            - absolute amp is within [0.9, 1.1]
+            - base is less than 0.1
+            - amp error is less than 0.1
+            - tau error is less than its value
+            - base error is less than 0.1
+        """
+        amp = fit_data.ufloat_params["amp"]
+        tau = fit_data.ufloat_params["tau"]
+        base = fit_data.ufloat_params["base"]
+
+        if amp.nominal_value > 0:
+            criteria = [
+                fit_data.reduced_chisq < 3,
+                abs(amp.nominal_value - 1.0) < 0.1,
+                abs(base.nominal_value) < 0.1,
+                curve.utils.is_error_not_significant(amp, absolute=0.1),
+                curve.utils.is_error_not_significant(tau),
+                curve.utils.is_error_not_significant(base, absolute=0.1),
+            ]
+        else:
+            # In SVD decomposition, the main vector is determined up to its sign.Therefore, two graphs
```

Comment:
I thought we said that you'd override `_format_data`, such that it will flip `ydata` if needed. The solution here is not good because the figures remain funny.